src/services/web-monitoring-api.js

```diff
@@ -77,6 +77,82 @@ export default class WebMonitoringApi {
             ));
     }
 
+    /**
+     * Add change information to the dictionary of insignificant changes in
+     * Google sheets.
+     *
+     * @param {Page}    page
+     * @param {Version} fromVersion
+     * @param {Version} toVersion
+     * @param {Object}  annotation
+     * @returns {Promise<boolean>}
+     */
+    addChangeToDictionary (page, fromVersion, toVersion, annotation) {
```

Comment:
Yeah, I didn’t bother because we intend to remove this functionality once analysts are fully transitioned over from spreadsheets. Might still be a good idea to, though. You’re are right; I was just being lazy again.